sztp-agent/pkg/dhcp/bootstrap_url.go

```diff
@@ -0,0 +1,25 @@
+/*
+SPDX-License-Identifier: Apache-2.0
```

Comment:
not sure I understand the need for this file here.... logic of what to use can/should be inside run-once in agent

Reply:
agree